x/validators/msg.go

```diff
@@ -21,10 +20,10 @@ func (*SetValidatorsMsg) Path() string {
 func (m ValidatorUpdate) Validate() error {
 	if len(m.Pubkey.Data) != 32 ||
 		strings.ToLower(m.Pubkey.Type) != "ed25519" {
-		return errors.WithCode(errInvalidPubKey, CodeInvalidPubKey)
+		return InvalidPubKeyErr
```

Comment:
Note that one of the main purposes of the new error package refactor (from my understanding, maybe not others) was to make it easy to define custom error strings inline, so we get more info.

Let's add the type in this error message. Since you don't want a custom function, can't we do:
`InvalidPubKeyErr.New(m.Pubkey.Type)`